client/app/components/Feed/Filter/Filter.jsx

```diff
@@ -1,12 +1,39 @@
-var React = require('react');
-
-var Filter = React.createClass({
-  render: function () {
-    return (
-      <div> Hello Super Sweet Filter
-      </div>
-    );
+import React from 'react';
+import getMuiTheme from 'material-ui/styles/getMuiTheme';
+import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
+
+import { Tabs, Tab } from 'material-ui/Tabs';
+import RaisedButton from 'material-ui/RaisedButton';
+import Subheader from 'material-ui/Subheader';
+
+import SvgIcon from 'material-ui/SvgIcon';
+import Earth from 'material-ui/svg-icons/maps/terrain';
+import Pin from 'material-ui/svg-icons/maps/place';
+import Personal from 'material-ui/svg-icons/maps/person-pin';
+
+let Filter = React.createClass({
```

Comment:
Why are we instantiating the class in this way? Is it in order to use the getInitialState function? 

`class ExampleComponent extends React.Component {
 constructor() {
  super();
  this. _handleClick = this. _handleClick.bind(this);
  this.state = Store.getState();
 }`

It looks like maybe we can just store the state in this.state